community/ssl/src/test/resources/org/neo4j/ssl/test-certificates/gen.sh

```diff
@@ -21,3 +27,9 @@ rm -rf ca
 ./gen-revocation.sh 04 3
 ./gen-revocation.sh 05 4
 ./gen-revocation.sh 06 5
+./gen-revocation.sh 01 6
```

Comment:
The first numbered column needs updating here to follow the pattern.

Reply:
Addressed